doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -12,10 +12,11 @@ Verified Version
 
 - Ubuntu version: **18.04**
 - GCC version: **7.5**
-- ACRN-hypervisor branch: **release_2.4 (v2.4)**
-- ACRN-Kernel (Service VM kernel): **release_2.4 (v2.4)**
+- ACRN-hypervisor branch: **release_2.5 (v2.5)**
+- ACRN-Kernel (Service VM kernel): **release_2.5 (v2.5)**
 - RT kernel for Ubuntu User OS: **4.19/preempt-rt (4.19.72-rt25)**
```

Comment:
Just to double-check: there was no change to that kernel version?

Reply:
Sorry, I didn't catch that... 
Compared with release_2.4 branch, release_2.5 branch maked some change.
The log are as follows.
root@automation:~/git/acrn-kernel# git log remotes/origin/release_2.5..remotes/origin/release_2.4 | grep commit
commit 5e4c2babd2b92c40af145c616d635f943746c6cf
commit 42a5e48c3e11f0172dc73573c7f7db639079977b
commit 2e6d4e94e8d69a6df628d9a12ab7f0500982d1a4
commit 3c65fcaa88aa6f72958878826a15f668bb40e6e3
commit 9ba917c2811b35c875ab8185c2da66a84cf4a962
commit f335b7610064bad0bdd285839fc7eb103b1af239
commit d198cab378f93689c1228d3d68d9e108a78058ae
commit 84a8471befd498e3e912c6cfeb0ce2d07409ed1b
commit 293747f2c62bbe8de94c25d66377aef5b7cf7224
commit 7c2e1c3aefff1beeb43f0595816055481efb046b
commit 1c69c53ee6a7afe176f7fd9932b6fc105c98f3bd
commit 906823618a8ebfd179a91d5150aa43bb52bea477
root@automation:~/git/acrn-kernel#

Reply:
Yes, we use the same version with release_2.4 for RT kernel.